pkgs/development/libraries/wlroots/protocols.nix

```diff
@@ -0,0 +1,38 @@
+{ lib, stdenv, fetchFromGitLab, pkg-config, wayland-scanner }:
+
+stdenv.mkDerivation rec {
+  pname = "wlr-protocols";
+  rev = "d998ee6fc64ea7e066014023653d1271b7702c09";
```

Comment:
Rev should be in src and the version should be unstable-dateofcommit

Reply:
I'm doing the wlr-protocols stuff in a separate PR: I'll apply your comments there (https://github.com/NixOS/nixpkgs/pull/147296) and rebase this PR once everything is in.